src/sites-configuration.ts

```diff
@@ -63,6 +63,17 @@ export const Sites: Record<string, DefaultSiteConfiguration> = {
       }
     }
   },
+    
+  iDeditor: {
+    link: "www.openstreetmap.org/edit",
+    paramOpts: [
+      { ordered: "#map={zoom}/{lat}/{lon}" },
+      { ordered: "?way/{osm_way_id}" },
+      { ordered: "?node/{osm_node_id}" },
+      { ordered: "?relation/{osm_relation_id}" },
+      { ordered: "?changeset/{osm_changeset_id}" },
```

Comment:
```suggestion
  ideditor: {
    link: "www.openstreetmap.org/edit",
    paramOpts: [
      { ordered: "#map={zoom}/{lat}/{lon}" },
      { ordered: "?way/{wayId}" },
      { ordered: "?node/{nodeId}" },
      { ordered: "?relation/{relationId}" },
      { ordered: "?changeset/{changesetId}" },
```
Hi, due to historic reasons, this file uses different names for the parameters

Reply:
Please add an entry `site_ideditor` in the file `addon/_locales/en/messages.json`. This is necessary to show an internationalizable title for this item